woocommerce/woocommerce-android
# [Better Errors] Redirect app password to authorization webview

Summary
==========
Fix issue #10802 by introducing the full login flow through the Application Password Tutorial screen. When the `continue` button is pressed there, the app will open the Application Password authorization through a web view and handle the auth data back to the Site credentials.

How to Test
==========
1. Make sure you have a store configured with something that can get in the way of the login, like the reCAPTCHA plugin.
2. Start the store login flow using the store address and direct user credentials.
3. Verify that the Username/Password screen redirects to the Application Password Tutorial screen.
4. Once in the tutorial, hit the `Continue` button and proceed to the Application Password authorization web view.
5. Once authorized, make sure the authentication flows completes as expected.

Update release notes:
